src/main/java/codeu/model/store/persistence/PersistentStorageAgent.java

```diff
@@ -21,12 +21,12 @@
 import java.util.List;
 
 /**
- * This class is the interface between the application and PersistentDataStore, which
- * handles interactions with Google App Engine's Datastore service. Currently this class simply
- * passes function calls through to PersistentDataStore, but this could be modified to make
- * asynchronous calls or to point to a different backend storage system.
+ * This class is the interface between the application and PersistentDataStore, which handles
+ * interactions with Google App Engine's Datastore service. Currently this class simply passes
+ * function calls through to PersistentDataStore, but this could be modified to make asynchronous
+ * calls or to point to a different backend storage system.
  *
- * This is a singleton; the single instance is accessed through getInstance().
+ * <p>This is a singleton; the single instance is accessed through getInstance().
```

Comment:
Is this correct? I thought it would be <p/>. Also, it might be clearer to move this to the blank line preceeding this line.

Reply:
github ate my comment...
I thought the `<p>` was supposed to be `<p/>`

Reply:
Yeah it looks weird, but I'm pretty sure `<p>` is correct. In fact Java 8's JavaDoc tool will give you an error if you use `<p/>` because technically it's not valid HTML (but, weirdly enough, a single unclosed `<p>` tag is valid). Sources: [here[(https://stackoverflow.com/questions/26049329/javadoc-in-jdk-8-invalid-self-closing-element-not-allowed) and [here](https://stackoverflow.com/questions/33787753/javadoc-closing-tags)

Reply:
OK. Special case. I get it.
I like the idea of putting it on a blank line by itself to better communicate intent. But it is up to you.

Reply:
If I move the `<p>` tag to its own line, the formatter actually "fixes" this by putting the `<p>` tag on the same line as the sentence that follows it. I'm going to err on the side of obeying the formatter.